### ISSUE TITLE
Typescript 3 support

### ISSUE THREAD

**Issue by ngehlert** [rubenv/angular-gettext-tools#192]
I know you are currently looking for a new maintainer, so just checking if you maybe are still in touch here.

It seems like the lib is somewhat outdated 
```
    "typescript": "~2.3.2",
    "typescript-eslint-parser": "^3.0.0"
```
the typescript-eslint-parser (newest version is 22.0.0) is also deprecated and should be replaced with `@typescript-eslint/parser`

Also I get some compile errors when running with TypeScript 3 (can't extract strings anymore from most of the TypeScript files)

Would you be willing to accept a PR with TS3 support and updated eslint-parser?

**Comment by rubenv** [comment]
Absolutely, feel free to submit one!

Thanks for pointing this out.

**Comment by ngehlert** [comment]
fixed with release v2.3.15
